### PR TITLE
DOCS-14048 mapReduce refactoring

### PR DESCRIPTION
**Pull request by jason-price-mongodb**

### CODE REVIEW

File: source/core/aggregation-pipeline.txt [hunk 1]
```diff
@@ -194,41 +194,29 @@ pipeline.
 Considerations
 --------------
 
-Sharded Collections
-~~~~~~~~~~~~~~~~~~~~
+Aggregation Pipeline as an Alternative to Map-Reduce
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (andf-mongodb) [inline source/core/aggregation-pipeline.txt:198]: • Same here, usage of agg pipelines on sharded cluster, limitations, and optimization seems more important than the MR replacement. Might this also be better suited as the last "Considerations" subheading?

**Review by andf-mongodb — COMMENTED** [review]
Hi Jason -- looks great overall, I just added some thoughts about a bit of IA for the two main agg landing pages. Let me know what you think! These suggestions are optional.

File: source/aggregation.txt [hunk 2]
```diff
@@ -87,22 +87,7 @@ internal optimization phase. See
...
-   see :doc:`/tutorial/map-reduce-examples`. See also
-   :doc:`/reference/map-reduce-to-aggregation-pipeline`.
-
-MongoDB also provides :doc:`map-reduce </core/map-reduce>` operations
-to perform aggregation. Map-reduce uses custom JavaScript functions to
-perform the map and reduce operations, as well as the optional
-*finalize* operation.
-
-.. include:: /images/map-reduce.rst
+.. include:: /includes/fact-use-aggregation-not-map-reduce.rst
```
[the remark this answers is not in the document]
Comment (jason-price-mongodb) [reply source/aggregation.txt:90]: Done.

File: source/core/aggregation-pipeline.txt (hunk 1 above)
Comment (jason-price-mongodb) [reply source/core/aggregation-pipeline.txt:198]: Done. Will rebuild stage and check changes.

**Pull request merged**